src/server/server.js

```diff
@@ -0,0 +1,45 @@
+const path = require('path');
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));
+
+const proxy = require('express-http-proxy'),
+  app = require('express')(),
+  cors = require('cors');
+
+let token;
+
+async function auth() {
+  if (token) return token;
+
+  let raw = JSON.stringify({
+    user: 'igorioha95',
+    password: 'Va1mXa610o7v'
+  });
+
+  let requestOptions = {
+    method: 'POST',
+    body: raw,
+    redirect: 'follow'
+  };
+
+  return fetch('https://pfa.foreca.com/authorize/token?expire_hours=2', requestOptions)
+    .then(response => response.json())
+    .then(result => (token = result.access_token))
+    .catch(error => console.log('error', error));
+}
+
+app.use(cors());
+
+app.use('/:toUrl', (req, res, next) =>
```

Comment:
1. you can just do app.:method:
2. '/:toUrl' is not the best fit. It doesn't tell me anything. You need to provide names for your API that will make sense to devs are they're using them

Reply:
I need to change name of param?

Reply:
I use proxy, that's why I write this param. I think that is more secure

Reply:
yes, you should rename it. Read about it here - https://rapidapi.com/guides/practices-naming-api-endpoints